### ISSUE TITLE
nvda 2015.4 is unable to run on windows vista

### ISSUE THREAD

**Issue by nikosdemetriou** [nvaccess/nvda#5570]
nvda 2015.4 when is installed on windows vista, when we try to run it the following message comes up.
[Window Title]
C:\Program Files\NVDA\nvda.exe

[Content]
C:\Program Files\NVDA\nvda.exe

A referral was returned from the server.

[OK]

**Comment by jcsteh** [comment]
Do you have all Windows updates installed? After some investigation, I believe this is very probably due to [the KB2763674 update](https://support.microsoft.com/en-us/kb/2763674) not being installed.

I was able to reproduce this on a Vista machine here that has never been updated. I'm just waiting for updates to apply to verify the above.

**Comment by jcsteh** [comment]
That said, I'm curious as to how you got NVDA 2015.4 installed in the first place. For me, even the installer simply won't run at present.

**Comment by nikosdemetriou** [comment]
Hi.
The installer was working but after install I couldn't open nvda.
The computer belongs to a friend and probably you are right I don't
think he has got the latest updates.

**Comment by nikosdemetriou** [comment]
I also forgot to tell you that all previous versions including 2015.3
work fine on his computer so probably it is something that changed the
last 3 months that stopped nvda of working as it should
Nikos

**Comment by jcsteh** [comment]
The certificate we use to sign and thus verify the authenticity of NVDA expired in October, so we had to switch to a new one. Unfortunately, without this update, Windows Vista can't handle the newer, more secure certificates. Given that this update has been available since 2012, it is reasonable to expect users to have updated.

**Comment by nikosdemetriou** [comment]
Ok thanks.
I will try and help my friend to apply the updates.

**Comment by jcsteh** [comment]
I confirmed that installing all Windows updates fixes this problem. There's nothing we can do to avoid this requirement, but there are some documentation changes we can make at least.

**Comment by nikosdemetriou** [comment]
Thanks a lot.
I will try and help him to install the updates needed.
Nikos

**Comment by bhavyashah** [comment]
Since all seems well and sorted, this ticket should be closed. @jcsteh

**Comment by Brian1Gaff** [comment]
Was this, in the end not because it needs a certain service pack in Vista to 
be installed?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
